buildSrc/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -177,13 +178,12 @@ abstract class DackkaPlugin : Plugin<Project> {
                     }
 
                     docsTask.configure {
-                        clientName.set(project.firebaseConfigValue { artifactId })
-                        // this will become useful with the agp upgrade, as they're separate in 7.x+
-                        val sourcesForKotlin = emptyList<File>()
+                        if (!isKotlin) dependsOnAndMustRunAfter(docStubs)
```

Comment:
isn't it cleaner to wire this dependency up via `javaSources.set(docStubs.flatMap { it.output })`?

Reply:
Yeah, but you'd be invoking docStubs for `kotlin` modules as well.

Reply:
sure, but it can be conditional right?

```kotlin
javaSources.set(if(isKotlin) sourcesForJava  else docStubs.flatMap { it.output })
```